### DISCUSSION TITLE
Why didn't you release latest build?

### DISCUSSION THREAD

**Discussion by SonaliBendre** [zhongfly/mpv-build#7] [Q&A] [answered]
This https://github.com/zhongfly/mpv-build/actions/runs/1445486190
Although I can download from Github Artifacts but just curious why you skip some, are they unstable?

**Comment by zhongfly** [comment, marked as answer]
The reason for this is that I hope to release at a relatively fixed time every day, and the content will not change after release. The CI build is mainly for people who want to try the latest version following the mpv update.
This does not involve the issue of whether the version is stable.

**Comment by zhongfly** [comment]
mpv is constantly updated, constantly adding new features, constantly discovering bugs and fixing them, and at the same time, other libraries it depends on are constantly updated. It can be considered that there will never be a stable version, just like there will never be a perfect thing. My suggestion is to download the latest version and use
